File: N1.5/N1.5.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter a string: ");
        String line = scanner.nextLine();
        int globalCount = 0;
        String finalRow = stringFixer(line, globalCount);
        System.out.println("\nFixed string: " + finalRow);
        System.out.println("Number of changes: " + globalCount);
    }

    public static String stringFixer(String line, int globalCount) {
        StringBuilder workRow = new StringBuilder(line);
        int count = 0;
        for (int i = 0; i < workRow.length() - 1; i++) {
            int codeSymbol = (int) workRow.charAt(i);
            codeSymbol++;
            char nextSymbol = (char) codeSymbol;
            if (workRow.charAt(i) == workRow.charAt(i + 1)) {
                count++;
            }
            if (count == 2) {
                workRow.insert(i + 1, nextSymbol);
                count = 0;
                globalCount++;
            }
        }
        return workRow.toString();
    }
}

File: N1.5/N.1.5.kt
import java.util.*

fun stringFixer(line: String): Pair<String, Int> {
    var workRow = line
    var addedCount = 0
    for (i in 0 until workRow.length - 1) {
        val codeSymbol = workRow[i].code
        val nextSymbol = (codeSymbol + 1).toChar()
        if (workRow[i] == workRow[i + 1]) {
            workRow = workRow.substring(0, i + 1) + nextSymbol + workRow.substring(i + 1)
            addedCount++ // Увеличиваем счетчик добавленных символов
        }
    }
    return Pair(workRow, addedCount)
}

fun main() {
    val scanner = Scanner(System.`in`)
    print("Enter row, please - ")
    val line = scanner.nextLine()
    val (finalRow, addedCount) = stringFixer(line)
    println("\nFixed row - $finalRow\nValue of changes - $addedCount")
}
